packages/ui/components/form/timezone-select/TimezoneSelect.tsx

```diff
@@ -64,6 +66,7 @@ export function TimezoneSelectComponent({
         ...(data ? addCitiesToDropdown(data) : {}),
         ...addCitiesToDropdown(cities),
       }}
+      onChange={({ value }) => setTimezone(value)}
```

Comment:
Aren't `onChange` and `onInputChange` handlers one and the same? The issue was that the timezone selector was disabled because of previously setting the `isDisabled` prop to true so is this really needed now?

Reply:
onInputChange is for the input box where you can type the timezone. Onchange is for change when you select an option. Anyways doing it directly on TimezoneSelect is not right as it would alter it for everywhere the component is used.